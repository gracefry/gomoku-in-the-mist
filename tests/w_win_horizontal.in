place A1
who
place H9
place B1
place G9
place B2
place E9
place B6
place F9
place H10
place I9
