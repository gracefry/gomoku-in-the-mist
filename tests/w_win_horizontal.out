W
White wins!
A1H9B1G9B2E9B6F9H10I9
Thank you for playing!
